src/components/GlobalStyle.tsx

```diff
@@ -755,14 +755,15 @@ const GlobalStyle: React.FC = () => {
         /* Anchor tag styles */
         /* Selected specifically for mdx rendered side icon link */
         .header-anchor {
-          position: relative;
+          position: relative !important;
```

Comment:
Due to some sort of build order, these styles are being overridden by the default styles that come with the plugin https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-autolink-headers/src/gatsby-ssr.js#L15 thats why I had to put those `!important`.

I'll see if I can revert these changes in the #7564 PR.